weather_mv/loader_pipeline/bq.py

```diff
@@ -136,6 +142,8 @@ def add_parser_arguments(cls, subparser: argparse.ArgumentParser):
         subparser.add_argument('--coordinate_chunk_size', type=int, default=10_000,
                                help='The size of the chunk of coordinates used for extracting vector data into '
                                     'BigQuery. Used to tune parallel uploads.')
+        subparser.add_argument('--disable_grib_schema_normalization', action='store_true', default=False,
+                               help="To disable grib's schema normalization. Default: off")
```

Comment:
One part of the documentation says "default: on", wheres here it's "off".

Reply:
As we have discussed, here in doc string of class we should specify the default value which represents actual behavior of the pipeline https://github.com/google/weather-tools/pull/184/files/4b2eb7b4b6e77bb374d1bd1361d7c050a9619b1f#diff-938d4c3999aa9d0bd9f832240de92dc8ea660bf71f8b9b6d751b3a16773ce4b1R97 not the default value of the flag.

Taking reference of : https://github.com/google/weather-tools/pull/184/files/4b2eb7b4b6e77bb374d1bd1361d7c050a9619b1f#diff-938d4c3999aa9d0bd9f832240de92dc8ea660bf71f8b9b6d751b3a16773ce4b1R92

Reply:
I see. I think I didn't understand fully in our discussion. It's made a bit more confusing since there's a double negative involved (disabling the feature is off).

For the second case (i.e. `Default: off`), how about we use more descriptive language? For example: "Default: normalization enabled". Then, it's more consistent and less confusing. WDYT?

Reply:
Sound good to me.